dask_geomodeling/raster/temporal.py

```diff
@@ -287,15 +294,40 @@ def _get_bin_start(dt, frequency, closed, label, timezone):
     return resampled.first().index[0]
 
 
-def _get_closest_label(dt, frequency, closed, label, timezone, side="both"):
+def _shift_bin_label(dt, frequency, timezone, n):
```

Comment:
To me using "label" and "bin" in the function name makes it sound more complicated to me. Something like "_shift_date" sounds appopriate enough to me?